### ISSUE TITLE
Leaf certs expired on January 27, 2022

### ISSUE THREAD

**Issue by daveyb** [brianshumate/vaultron#27]
Hey @brianshumate - thanks for sharing Vaultron with the world - it's a great dev tool!

## Expected Behavior
`./form` brings up a working Vaultron vault cluster as it typically does 😄 

## Actual Behvior
The leaf certs for both raft and consul-flavored deployments expired `Jan 27 13:43:20 2022 GMT`, so Vaultron Vault clusters will not start.

This includes the certs found in `black_lion/tls/`, `red_lion/tls/`, and `yellow_lion/tls/` directories.

## Steps to Reproduce
1. I first did this with a raft-flavored cluster, so `TF_VAR_vault_flavor=raft` in my shell env.
1. Run `./form`
1. Run `./ion_darts` to export configs for this vaultron instance
1. Run `vault status` to see status of cluster:
    ![Screen Shot 2022-01-27 at 11 18 38 AM](https://user-images.githubusercontent.com/9647/151399265-3094e91e-759e-40a8-8fd5-015082085424.png)
1. Run `docker logs vaultron-vault0` node to see what's up:
    ![Screen Shot 2022-01-27 at 11 19 34 AM](https://user-images.githubusercontent.com/9647/151399553-7ccfba6d-3cb4-4955-bdb2-4746408dbef0.png)
1. Visit https://127.0.0.1:8200 in the browser:
    ![Screen Shot 2022-01-27 at 11 20 41 AM](https://user-images.githubusercontent.com/9647/151399655-debdc225-ce4f-4be3-9eff-091366f1e3fd.png)

## View leaf certs using openssl
1. Run `openssl x509 -in black_lion/tls/vault-server-0.crt -noout -text` on an example leaf cert to see it expired today:
    ```
    Signature Algorithm: sha256WithRSAEncryption
        Issuer: CN=node.arus.consul
        Validity
            Not Before: Oct 25 13:42:50 2019 GMT
            Not After : Jan 27 13:43:20 2022 GMT
    ```
> NOTE: the certs in `etc/tls/` are NOT expired, just the leaf certs in the `*_lion/tls` directories.

**Comment by brianshumate** [comment]
Thanks for the report, I'll try to get these updated soon!

**Comment by brianshumate** [comment]
Resolved in latest release (3.9.0), thanks again for the heads up!
